tap_postgres/db.py

```diff
@@ -131,6 +132,10 @@ def selected_value_to_singer_value_impl(elem, sql_datatype):
         cleaned_elem = str(elem)
     elif isinstance(elem, str):
         cleaned_elem = elem
+    elif sql_datatype == 'string':
+        cleaned_elem = (elem.tobytes()).decode('utf-8')
```

Comment:
For a bytea column, this means we can set it's type to `string` and it'll be converted like this?  Have you compared this conversion to string with what we do elsewhere?  I think in SQL, we use decode to 'hex'.

This will be applied to existing strings too - does it work?